CHANGELOG.md

```diff
@@ -2,3 +2,11 @@ Changes by Version
 ==================
 
 <!-- next version -->
+
+## 0.1.0
+
+### 🚀 New components 🚀
+
+- `operator`: Initial release of tempo operator ()
+  - Supports tempo 2.0.1
```

Comment:
It would be great to add a link to Tempo - a similar like we do it in the OTEL operator